static/app/views/dashboardsV2/create.tsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import {browserHistory, PlainRoute, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import {createDashboard} from 'app/actionCreators/dashboards';
+import {addSuccessMessage} from 'app/actionCreators/indicator';
+import {Client} from 'app/api';
+import Feature from 'app/components/acl/feature';
+import Alert from 'app/components/alert';
+import LightWeightNoProjectMessage from 'app/components/lightWeightNoProjectMessage';
+import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
+import SentryDocumentTitle from 'app/components/sentryDocumentTitle';
+import {t} from 'app/locale';
+import {PageContent} from 'app/styles/organization';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+
+import Controls from './controls';
+import Dashboard from './dashboard';
+import {DEFAULT_STATS_PERIOD, EMPTY_DASHBOARD} from './data';
+import DashboardTitle from './title';
+import {DashboardDetails, DashboardState, Widget} from './types';
+import {cloneDashboard} from './utils';
+
+const UNSAVED_MESSAGE = t('You have unsaved changes, are you sure you want to leave?');
+
+type Props = {
+  api: Client;
+  organization: Organization;
+  route: PlainRoute;
+} & WithRouterProps<{orgId: string; dashboardId: string}, {}>;
+
+type State = {
+  dashboard: DashboardDetails;
+  dashboardState: DashboardState;
+};
+
+class CreateDashboard extends React.Component<Props, State> {
```

Comment:
since we are not using `AsyncComponent` and this is a new component, should we update it to be a functional component and use hooks?